src/main/java/bookface/logic/parser/LoanCommandParser.java

```diff
@@ -19,8 +19,7 @@
  */
 public class LoanCommandParser implements Parseable<LoanCommand> {
 
-    //TODO Check if there's a better way to detect invalid loan commands
-    public static final String VALIDATION_REGEX = "(\\d+\\s+\\d+\\s+(?s).*)|(\\d+\\s+\\d+)";
+    public static final String VALIDATION_REGEX = "^\\d+\\s+\\d+$";
 
```

Comment:
Why should this be changed as `(\\d+\\s+\\d+\\s+(?s).*)` checks for `2 3 3 weeks later` for example?

Reply:
You're right, I'll revert it